.github/workflows/build.yml

```diff
@@ -153,7 +153,7 @@ jobs:
           GPG_PASSWORD: ${{ secrets.GPG_PASSWORD }}
 
   required-status-check:
-    if: github.event_name == 'pull_request'
+    if: github.event_name == 'pull_request' && always()
```

Comment:
I'm not following how this helps. Isn't `&& always()` basically `&& true` which is always true?

Reply:
one would think...

check out https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/evaluate-expressions-in-workflows-and-actions#always

and you can see the difference below, `required-status-check` is run and fails (and so should block merging even after someone approves this PR), whereas in #1542 for example, `required-status-check` is skipped (and therefore didn't block merging)